### PR TITLE
[MIG] partner_employee_quantity: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
- Minor migration tweaks

cc @Tecnativa

### CODE REVIEW

**Review by yajo — APPROVED** [review]
Code OK, please squash by logical changeset/author.

File: partner_employee_quantity/README.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Images
+------
+
+* `Antiun Ingeniería S.L. <http://www.antiun.com>`_: Icon.
+
+Contributors
+------------
+
+* Antonio Espinosa <antonioea@antiun.com>
```
Comment (rafaelbn) [inline partner_employee_quantity/README.rst]: antonio.espinosa@tecnativa.com

File: partner_employee_quantity/README.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* Rafael Blasco <rafael.blasco@tecnativa.com>
+* Jairo Llopis <yajo.sk8@gmail.com>
```
Comment (rafaelbn) [inline partner_employee_quantity/README.rst]: jairo.llopis@tecnativa.com

File: partner_employee_quantity/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# © 2015 Tecnativa S.L. - Jairo Llopis
+# © 2017 Tecnativa S.L. - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Employee quantity in partners',
+    'summary': 'Know how many employees a partner has',
+    'version': '10.0.1.0.0',
+    'category': 'Customer Relationship Management',
+    "author": "Antiun Ingeniería S.L., Odoo Community Association (OCA)",
```
Comment (rafaelbn) [inline partner_employee_quantity/__manifest__.py]: Add Tecnativa

File: partner_employee_quantity/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "license": "AGPL-3",
+    "application": False,
+    'website': 'http://www.antiun.com',
```
Comment (rafaelbn) [inline partner_employee_quantity/__manifest__.py]: www.tecnativa.com

**Review by rafaelbn — CHANGES_REQUESTED** [review]

**Comment by chienandalu** [comment]
@rafaelbn You review now to rebase and merge

**Review by rafaelbn — APPROVED** [review]

**Comment by chienandalu** [comment]
Not sure of why Travis is failing

**Comment by yajo** [comment]
It's #434

**Comment by pedrobaeza** [comment]
Please rebase to get a green branch

**Pull request merged**